pom.xml

```diff
@@ -348,14 +350,50 @@ under the License.
         </profile>
 
         <profile>
-            <id>scala-2.13</id>
+            <id>spark3</id>
+            <modules>
+                <module>paimon-spark/paimon-spark3-common</module>
+                <module>paimon-spark/paimon-spark-3.5</module>
+                <module>paimon-spark/paimon-spark-3.4</module>
+                <module>paimon-spark/paimon-spark-3.3</module>
+                <module>paimon-spark/paimon-spark-3.2</module>
+            </modules>
             <properties>
+                <scala.binary.version>2.12</scala.binary.version>
+                <scala.version>${scala212.version}</scala.version>
+                <paimon-spark-common.spark.version>3.5.3</paimon-spark-common.spark.version>
+                <paimon-sparkx-common>paimon-spark3-common</paimon-sparkx-common>
```

Comment:
I prefer to `paimon-spark-common` . cc @JingsongLi

Reply:
paimon-spark-common may cause ambiguity with the existing `paimon-spark-common` module, I prefer to distinguish them

Reply:
I am OK with `paimon-sparkx-common`.